lib/search/response.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'common/client/concerns/service_status'
+require 'common/models/base'
+
+module Search
+  class ResultsResponse < Common::Base
+    include Common::Client::ServiceStatus
+
+    attribute :status, Integer
+    attribute :body, Hash
+
+    def initialize(status, attributes = nil)
+      super(attributes) if attributes
+      self.status = status
+    end
+
+    def self.from(response)
+      body = response.body
+      new(response.status, body: body)
+    end
+
+    def ok?
+      status == 200
+    end
+
+    def cache?
```

Comment:
curious about the dup of ok? and cache? are both needed for context .. would alias make more sense

Reply:
Yeah, this sets up the API to be cachable with our internal framework.  Good point.  I believe framework has moved away from the `ok?` method in favor of the `cache?` method, so we could just move the `status == 200` into `cache?` and remove `ok?`.